core/src/main/scala/neotypes/internal/utils.scala

```diff
@@ -34,4 +37,82 @@ private[neotypes] object utils {
 
     loop(acc = factory.newBuilder)
   }
+  def createQuery(queryData: Either[String, QueryArg]*): DeferredQueryBuilder = {
```

Comment:
We should reuse these utils as well on the **Scala 2** version of the macro.

Reply:
Done.